contracts/eosiolib/db.h

```diff
@@ -229,7 +229,7 @@ int32_t upper_bound_i64( account_name scope, account_name code, table_name table
  *
  *  @return 1 if a record was removed, and 0 if no record with key was found
  */
-int32_t remove_i64( account_name scope, table_name table, void* data );
+int32_t remove_i64( account_name scope, table_name table, account_name bta, void* data );
```

Comment:
Do we need bta for remove?  are there any cases where this value would be something different than the one present on the existing record?

Reply:
Deleted bta from remove.